physicsnemo/distributed/manager.py

```diff
@@ -356,7 +356,14 @@ def initialize_slurm(port):
         rank = int(os.environ.get("SLURM_PROCID"))
         world_size = int(os.environ.get("SLURM_NPROCS"))
         local_rank = int(os.environ.get("SLURM_LOCALID"))
-        addr = os.environ.get("SLURM_LAUNCH_NODE_IPADDR")
+        try:
+            addr = os.environ.get("SLURM_LAUNCH_NODE_IPADDR")
+        except TypeError:
```

Comment:
Sorry @Tristanjmeyers - I looked at this more closely and we need to adjust this.  `os.environ.get` will return `None` for missing variables, and raise no error. I think you can either:
- Remove the try/except type error, and change to `if add is None: raise EnvErr...`; or
- change from `os.environ.get()` to `os.environ['SLURM_LAUNCH_NODE_IPADDR'] and `TypeError` to `KeyError`.